cypress/integration/pages/frontPage/tests.js

```diff
@@ -18,7 +18,16 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
           serviceHasPublishedPromo(service) &&
           Cypress.env('APP_ENV') === 'live'
         ) {
-          it('individual promo should link to corresponding article pages and back navigation should link to frontpage', () => {
+          it('individual promo should link to corresponding article pages and back navigation should link to frontpage', done => {
+            // This is to catch an application error that keeps failing live E2Es
+            // We are waiting for a response from Google to find a fix
+            // And in the meantime are stopping this error failing the tests
+            // eslint-disable-next-line no-unused-vars
```

Comment:
@LilyL0u could you link the issue you were going to create in the comment, just will make it easier to resolve this if it gets forgotten and someone comes across this later.

Also, nice work getting this to work in a Single test, I gave up and did it globally in the example I send you this is much more focused 👍

Reply:
Linked. Thanks :)

Reply:
Sorry, I meant in the code comment, did you forget to push?

Reply:
OH i see what you mean

Reply:
I thought you meant a github comment. I have now added the comment in the code.

Reply:
Thanks @LilyL0u